runtime/oti/j9nonbuilder.h

```diff
@@ -5043,12 +5043,18 @@ typedef struct J9VMThread {
 
 #if defined(OMR_GC_COMPRESSED_POINTERS)
 #if defined(OMR_GC_FULL_POINTERS)
+/* Mixed mode - necessarily 64-bit */
 #define J9VMTHREAD_COMPRESS_OBJECT_REFERENCES(vmThread) (0 != (vmThread)->compressObjectReferences)
+#define J9VMTHREAD_REFERENCE_SHIFT(vmThread) (J9VMTHREAD_COMPRESS_OBJECT_REFERENCES(vmThread) ? 2 : 3)
```

Comment:
Doesn't the shift depend on the heap size?  Is there a named constant we can use rather than hardcoded 2 / 3?

Reply:
2 and 3 are log2() of slot size in bytes, 4 and 8 correspondently. Obviously does not depend of heap size

Reply:
I thought we had different shift amounts based on the heap size.  Didn't you do a bunch of work previously to allow a larger shift when using heaps > 50-ish gigs?

Reply:
Dmitri straightened me out - this is not the same as the heap based shifts.  @gacholio Can you rename this to make it clearer this a different shift?

Reply:
I agree the name isn't great, but I've already used the same naming in OMR, so I'd rather be consistent than pretty (which isn't going to happen in my case, let's face it).